pkg/machine/qemu/machine.go

```diff
@@ -306,6 +307,10 @@ func (v *MachineVM) Init(opts machine.InitOptions) (bool, error) {
 		target := source
 		readonly := false
 		securityModel := "mapped-xattr"
+		if runtime.GOOS == "darwin" {
+			// see https://github.com/containers/podman/discussions/16102
+			securityModel = "none"
```

Comment:
IMO this should be consistent between mac and linux, so i would remove the runtime check here.